### ISSUE TITLE
[Steam Reviews Viewer] Setup/getting started query (documentation request? but might be bug)

### ISSUE THREAD

**Issue by Kookiness** [darklinkpower/PlayniteExtensionsCollection#521]
### Extension name

Steam Reviews Viewer

### Feature Description

I'm probably missing something, but I cannot find a way to use the features to view Steam game reviews or videos. Not sure if this is a bug or ought to be a request for more documentation in case I'm doing something wrong.

Steam Reviews Viewer is installed and enabled, and appears in context menu when clicking a game.

I have left-clicked on various different Steam games in Playnite, in the Details view, and tried manually downloading the review data for them, and restarting Playnite, but no reviews or videos appear when a game is clicked.

I didn't find an answer in the Help/Troubleshooting link: https://github.com/darklinkpower/PlayniteExtensionsCollection/wiki/Review-Viewer

I've included a screenshot of what I see in Playnite, in case it helps.

Any guidance on what I'm missing would be greatly appreciated. Thank you.

### Screenshots

![image](https://github.com/darklinkpower/PlayniteExtensionsCollection/assets/75261412/621663c2-f4cd-4815-881c-7a774170aa73)

**Comment by darklinkpower** [comment]
You need to use a theme that supports the extension, for example, the themes I develop: https://github.com/darklinkpower/PlayniteExtensionsCollection?tab=readme-ov-file#themes

Videos are available in right click > videos but if you want videos in the main window you need to use the Extra Metadata Loader extension: https://playnite.link/addons.html#ExtraMetadataLoader_705fdbca-e1fc-4004-b839-1d040b8b4429

**Comment by Kookiness** [comment]
Many thanks for the guidance.

I can't see anything about videos when I right-click a game:

![image](https://github.com/darklinkpower/PlayniteExtensionsCollection/assets/75261412/68c7fde9-bff7-4dd3-88f4-23bd2daa4bbe)

To clarify, I was using the theme that Playnite comes with by default. Are you saying your addon is not compatible with that?

As per your advice, I have installed your Harmony theme, and can still not find Steam reviews displayed in Playnite:

![image](https://github.com/darklinkpower/PlayniteExtensionsCollection/assets/75261412/33bb53d0-6985-4a50-b8c3-6d0766433908)

**Comment by Kookiness** [comment]
Since this is unresolved despite following @darklinkpower's instructions, would it be better for me to open a new issue and report it as a bug?

**Comment by Kookiness** [comment]
Never mind, I found the "Reviews" tab. Shame I couldn't find any instructions to tell me that's what I needed to look for.

The Positive/Negative filter feature is showing "NaN%" though, so I think it might be broken (I suppose that's worth a new issue):

![Uploading image.png…]()

**Comment by Kookiness** [comment]
Okay, I found the "Reviews" tab. I couldn't find any instructions to tell me that's what I needed to look for. I still don't see anything about videos when right-clicking, but the Extra Metadata Loader works to show them, thank you.

The Positive/Negative filter feature on the Reviews tab is showing "NaN%", so I think it might be broken (I suppose that could be worth a new issue):

![image](https://github.com/darklinkpower/PlayniteExtensionsCollection/assets/75261412/b45480ab-d813-4164-8fba-1b397d6dd285)

Enabling the Positive/Negative filter switches back to the Description tab. Not sure if that counts as a bug, but it seems like it might.
